contributors/design-proposals/customresources-validation.md

```diff
@@ -306,6 +310,22 @@ The following example illustrates how a schema can be used in `CustomResourceDef
 }
 ```
 
+### Error messages
+
+The following examples illustrate the type of validation errors generated by using the go-openapi validate library.
+
+1. `data.foo in body should be at least 4 chars long`
+2. `data.foo in body should be greater than or equal to 10`
+3. `data.foo in body should be one of [bar baz]`
+4. `data.foo in body must be of type integer: "string"`
```

Comment:
No, string is the "type". It does not use the description field while generating errors. :confused: